metals/src/main/scala/scala/meta/internal/metals/logging/package.scala

```diff
@@ -0,0 +1,32 @@
+package scala.meta.internal.metals
```

Comment:
if this package object is toplevel then it'll be possible to use just `logging` not matter where it will be used 🤔

Reply:
Not sure, what do you mean?